lib/tests/streamlit/runtime/app_session_test.py

```diff
@@ -68,6 +71,16 @@ def _create_test_session(event_loop: Optional[AbstractEventLoop] = None) -> AppS
     MagicMock(spec=LocalSourcesWatcher),
 )
 class AppSessionTest(unittest.TestCase):
+    def setUp(self) -> None:
+        super().setUp()
+        media_file_manager._media_file_manager = MediaFileManager(
+            MemoryMediaFileStorage("/mock/media")
+        )
```

Comment:
Lots of tests rely on the MediaFileManager singleton existing, so we create it in this base class.